test/e2e/storage/pd.go

```diff
@@ -452,28 +455,15 @@ func countReadyNodes(c clientset.Interface, hostName types.NodeName) int {
 
 func verifyPDContentsViaContainer(f *framework.Framework, podName, containerName string, fileAndContentToVerify map[string]string) {
 	for filePath, expectedContents := range fileAndContentToVerify {
-		var value string
-		// Add a retry to avoid temporal failure in reading the content
-		for i := 0; i < maxReadRetry; i++ {
-			v, err := f.ReadFileViaContainer(podName, containerName, filePath)
-			value = v
-			if err != nil {
-				framework.Logf("Error reading file: %v", err)
-			}
-			framework.ExpectNoError(err)
-			framework.Logf("Read file %q with content: %v (iteration %d)", filePath, v, i)
-			if strings.TrimSpace(v) != strings.TrimSpace(expectedContents) {
-				framework.Logf("Warning: read content <%q> does not match execpted content <%q>.", v, expectedContents)
-				size, err := f.CheckFileSizeViaContainer(podName, containerName, filePath)
-				if err != nil {
-					framework.Logf("Error checking file size: %v", err)
-				}
-				framework.Logf("Check file %q size: %q", filePath, size)
-			} else {
-				break
-			}
+		// No retry loop as there should not be temporal based failures
+		v, err := f.ReadFileViaContainer(podName, containerName, filePath)
```

Comment:
actually what if there is a temporary failure in `kubectl exec` - network issue or something

Reply:
It internally retries: https://github.com/kubernetes/kubernetes/blob/d6a3518ea2ad92fb48e1de6c10f1fbd7e61736c3/test/e2e/framework/framework.go#L496